config.json

```diff
@@ -134,10 +134,10 @@
             "MsBuildLogging":"/flp:v=normal;LogFile=build-packages.log"
           }
         },
-        "signpackages": {
+        "signpkgs": {
```

Comment:
Just so I understand how did this confuse run.cmd?

Reply:
When there was both a setting & an alias named `signpackages` (case insensitive), run.cmd defaulted to running the setting (i.e. the target), and not the alias. So it would pass `t/:SignPackages`, but not run publish.msbuild. I'm thinking it goes top-down and runs the first thing with a matching name.